CHANGELOG.md

```diff
@@ -105,6 +105,28 @@
 
 ### Language Server
 
+- The Language Server can now suggest a code action to fill in the missing
+  patterns of a case expression:
+
+  ```gleam
+  case #(True, False) {
```

Comment:
Could you use a different example please 🙏  We don't want to recommend using tuples like this